src/components/header.js

```diff
@@ -21,26 +22,29 @@ function Header() {
     if (path !== '/')
       return (
         <OrangeFCCLogo
-          className="logo"
+          className={componentStyles.logo}
           alt={`${site.siteMetadata.title} Logo`}
         />
       );
     return (
-      <BlueFCCLogo className="logo" alt={`${site.siteMetadata.title} Logo`} />
+      <BlueFCCLogo
+        className={componentStyles.logo}
+        alt={`${site.siteMetadata.title} Logo`}
+      />
     );
   };
 
   return (
     <header className="bg-white">
-      <div className="navBar">
+      <div className={componentStyles.navBar}>
         <Link to="/">{logo(currentPathname)}</Link>
 
         <button
-          className="navMenuButton"
+          className={componentStyles.navMenuButton}
           onClick={() => toggleExpansion(!isExpanded)}
         >
           <svg
-            className="burgerIcon"
+            className={componentStyles.burgerIcon}
```

Comment:
This is neat. It's me helping learn how to style. I just need to dig in and see if it's a react, tailwind, or gatsby convention being used here.

Reply:
Hmmmm, I'm not entirely certain but I believe that's a webpack feature using the css-loader plugin:
https://webpack.js.org/loaders/css-loader/

It came prepackaged with the template we started with.